flink-table/flink-table-planner/src/test/resources/org/apache/flink/table/planner/plan/rules/logical/subquery/SubQueryAntiJoinTest.xml

```diff
@@ -1833,8 +1833,8 @@ LogicalProject(b=[$1])
       :  +- LogicalJoin(condition=[=($0, $5)], joinType=[left])
       :     :- LogicalJoin(condition=[true], joinType=[inner])
       :     :  :- LogicalTableScan(table=[[default_catalog, default_database, l, source: [TestTableSource(a, b, c)]]])
-      :     :  +- LogicalAggregate(group=[{}], c=[COUNT()], ck=[COUNT($0)])
-      :     :     +- LogicalProject(EXPR$0=[1])
+      :     :  +- LogicalProject(c=[$0], ck=[$0])
```

Comment:
Is this also tested somewhere with IT cases? We should make sure that ExecNodes are still working properly.

Reply:
The same reason as mentioned in comment above:
added optimization in Calcite to remove args for nonDistinct SqlAggCountFunction and successors